opentelemetry-sdk/src/opentelemetry/sdk/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .version import __version__
```

Comment:
Why do you need this?

Reply:
I was thinking about troubleshooting scenario, where we want to see which version of the SDK customers are using, we do need to distinguish SDK versions versus API versions.
For example, if the queue for traces are full, I'd expect we log something like:
```
2019-06-27T22:36:59Z OpenTelemetry API loaded (version=1.0.0)
2019-06-27T22:37:15Z OpenTelemetry SDK loaded (version=1.0.3)
2019-06-27T22:38:30Z traces queue full, started to drop spans
2019-06-27T22:38:50Z traces queue recovered
```

Reply:
I mean why do you need to import it here instead of in whatever module uses it (the trace queue in your example)?

Reply:
I see, my intent was to save some typing, so we can import `sdk.__version__` instead of `sdk.version.__version__`. Later we can probably rename `version.py` to `package_info.py` if we have more stuff to put here.